client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js

```diff
@@ -79,17 +79,22 @@ export const getTask = (
 		case CHECKLIST_KNOWN_TASKS.EMAIL_VERIFIED:
 			taskData = {
 				timing: 1,
-				title: translate( 'Confirm your email address' ),
+				title: (
+					<>
+						{ translate( 'Confirm your email address' ) }
+						<span className="site-setup-list__nav-item-email">{ userEmail }</span>
+					</>
+				),
 				description: translate(
-					'Please click the link in the email we sent to %(email)s. ' +
-						'Typo in your email address? {{changeButton}}Change it here{{/changeButton}}.',
+					'We have sent an email to this address to verify your account. Not in inbox or spam folder? Tap the Resend email button! ' +
+						'{{emailWrapper}}%(email)s{{/emailWrapper}} {{changeButton}}Change{{/changeButton}}',
```

Comment:
Noting that as the email and change button parts are at the end of the string, they're making it needlessly more complex for the translators. I'd suggest moving them out of the string to simplify the localization for translators.

Reply:
Thanks @tyxla I have removed the components from appearing within the translation function, going forward I will remember not to do this. Great to be back from holiday!